src/sdk/model/company/__json__/currency-type.ts

```diff
@@ -0,0 +1,5 @@
+export type Currency = 'AUD' |
```

Comment:
we already have this enum in common/billing

Reply:
👍